src/classes/HiFiCommunicator.ts

```diff
@@ -248,11 +256,13 @@ export class HiFiCommunicator {
             let webRTCSignalingAddress = `wss://${signalingHostURLSafe}:${signalingPort}/?token=`;
             this._mixerSession.webRTCAddress = `${webRTCSignalingAddress}${hifiAuthJWT}`;
 
-            HiFiLogger.log(`Using WebRTC Signaling Address:\n${webRTCSignalingAddress}<token redacted>`);
+            HiFiLogger.log(`Using WebRTC Signaling Address:
```

Comment:
was this change on purpose? if so, what is it for?

Reply:
Yep, this is because the "\n"s that were embedded in the template literals (the backticked strings) made my console logs full of explicit "\n"s instead of ACTUAL linebreaks, and it has been making me crazy and it's really hard for me to debug. If there's a way to get the console output to output an _actual_ newline instead of a "\n" when "\n" is embedded inside a template literal, I'm happy to use that instead and go back to "\n"s. Otherwise, if we're using template literals, we should probably be using literal linebreaks rather than "\n"s. (Or we could stop using template literals, which I'm also OK with because TBH I don't like them -- backtick is a very special character for bash and others, and I'd rather use real quotes for JavaScript strings. :-) But I understand that JS programmers like 'em!)